hack/unwanted-dependencies.json

```diff
@@ -181,15 +174,17 @@
         "github.com/Microsoft/hcsshim"
       ],
       "golang.org/x/exp": [
+        "github.com/Microsoft/hcsshim",
         "github.com/antlr4-go/antlr/v4",
+        "github.com/cilium/ebpf",
```

Comment:
adding more deps on an explicitly unstable library is *really* unfortunate ... can we determine what experimental bits they are using and proposal upstream changes to drop them out?

Reply:
https://github.com/cilium/ebpf/issues/1095 
We have opened an issue for this before.